numpy/distutils/ccompiler_opt.py

```diff
@@ -295,6 +296,10 @@ class _Config:
             group="AVX512VBMI2 AVX512BITALG AVX512VPOPCNTDQ",
             detect="AVX512_ICL", implies_detect=False
         ),
+        AVX512_SPR = dict(
+            interest=46, implies="AVX512_ICL AVX512FP16",
+            detect="AVX512_SPR", implies_detect=False
+        ),
```

Comment:
```suggestion
        AVX512_SPR = dict(
            interest=46, implies="AVX512_ICL", group="AVX512FP16"
            detect="AVX512_SPR", implies_detect=False
        ),
```
attr group will enable #def `NPY_HAVE_AVX512FP16` without providing a target for it within `@targets`.

Reply:
fixed.